.github/pr-welcome-community.md

```diff
@@ -0,0 +1,38 @@
+## 👋 Welcome to PyAirbyte!
+
+Thank you for your contribution from **{{ .repo_name }}**! We're excited to have you in the Airbyte community.
+
+### Testing This PyAirbyte Version
+
+You can test this version of PyAirbyte using the following:
+
+```bash
+# Run PyAirbyte CLI from this branch:
+uvx 'git+https://github.com/{{ .repo_name }}.git@{{ .branch_name }}' pyab --help
```

Comment:
I don't think this syntax is correct. When the package name (airbyte) doesn't match the CLI name (pyab or pyairbyte), then I think we need --from.